docs/sphinx-docs/source/conf.py

```diff
@@ -107,6 +107,8 @@
                     "*sas.sasgui.guiframe.custom_pstats.rst", # pstats not in standard library on Ubuntu out of the box
                     ]
 
+autodoc_mock_imports = ['sip', 'PyQt5', 'PyQt5.QtGui', 'PyQt5.QtCore', 'PyQt5.QtWidgets']
+
```

Comment:
From the [autodoc_mock_imports doc](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_mock_imports):

    You may only specify the root package of the dependencies themselves and omit the sub-modules

So `['sip', 'PyQt5']` should be enough.

Reply:
Potentially. The docs aren't actually built on CI so I'll have to manually test that.

(Perhaps an additional PR would be welcome to build the docs on CI too so that things like #1395 and #1396 get caught earlier.)

Reply:
Yes, this should be a separate ticket.  We are already building them on the Jenkins server, but the doc errors aren't visible unless you dig into the console output from the build.

Maybe better [setuptools integration](https://www.sphinx-doc.org/en/master/usage/advanced/setuptools.html) would help?    Our doc builds are a little complicated since we are pulling in sasmodels and bumps documentation parts, so existing tools would require tweaking.